mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <!-- Generic Strings -->
```

Comment:
@gabrielluong  @mcarare: For all the strings under "Generic Strings", we will need a string comment for each, so localizers know what they correspond to.
In fact, depending on the language, the context will affect how these words are translated to.
I've reviewed the rest and it looks good, so once this is fixed I can expose these for translation. thanks